.github/workflows/renovate.yml

```diff
@@ -13,11 +14,29 @@ on:
   schedule:
     - cron: '3 * * * *'
 jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+
+      - name: Validate Renovate config
+        uses: renovatebot/github-action@v43.0.10
+        env:
+          RENOVATE_DRY_RUN: lookup
+          LOG_LEVEL: debug
```

Comment:
We are using log_level as `info` for renovate job. Should we keep that consistent throughout?

Reply:
Probably? I have no strong opinion on this...feel free to tweak!